ebean-core/src/main/java/io/ebeaninternal/api/SpiQuery.java

```diff
@@ -92,6 +92,11 @@ enum Type {
      */
     ATTRIBUTE(FIND_ATTRIBUTE, "findAttribute", false, false),
```

Comment:
This all looks fine.

I'm putting a comment here because it looks like we overload this exisint ATTRIBUTE to mean both findSingleAttribute and findSingleAttributeList ... and that is ok only because we don't have query caching for findSingleAttribute [only on findSingleAttributeList].

So this comment is here because I'm noting that. PR looks good.